Gopkg.lock

```diff
@@ -154,12 +162,14 @@
     "ext",
     "log"
   ]
-  revision = "1361b9cd60be79c4c3a7fa9841b3c132e40066a7"
+  revision = "1949ddbfd147afd4d964a9f00b24eb291e0e7c38"
+  version = "v1.0.2"
 
 [[projects]]
   name = "github.com/paulbellamy/ratecounter"
   packages = ["."]
-  revision = "5a11f585a31379765c190c033b6ad39956584447"
+  revision = "524851a93235ac051e3540563ed7909357fe24ab"
```

Comment:
This revision went backwards but we only lost a test coverage commit so it shouldn't matter.

Reply:
I'm going to switch the others to follow the master branch and keep this one the same. I would rather move us to using versions if possible.